### DISCUSSION TITLE
Lines appearing in form

### DISCUSSION THREAD

**Discussion by jellyo-o** [twbs/bootstrap#37429] [Q&A]
Hi, I am using bootstrap to create a login system and I came across this issue where scaling the container (bigger or smaller) makes these lines appear in the input form. I have no idea how to remove them. In my CSS I use ` form {scale: 0.9}` to make the form smaller. Is there a way to remove these lines or is there another way I can make the form smaller? 

![image](https://user-images.githubusercontent.com/113155652/200050655-fb7caa0c-d012-43fc-bc54-39ddcd22a471.png)

**Comment by tbolt** [comment]
Just checking, you are using `transform`? Like-
```
.scaled {
  transform: scale(0.11);
}
```
I'm not sure what's causing that but you may be better off using a `max-width` and `max-height` along with a larger/smaller font-size.

**Reply by jellyo-o** [reply]
yea I scale them like that and those weird lines appear. How should I go about using max-width/height and font-size? I'm not too familiar with bootstrap sizing. I'm looking to scale the whole form container (i.e the "Log in" title to the "register now" text at the bottom, it's one whole div)
